ptsprojects/mynewt/gap_wid.py

```diff
@@ -98,6 +116,12 @@ def hdl_wid_204(desc):
     return btp.check_discov_results(addr_type=0x02)
 
 
+def hdl_wid_227_force_pair(desc):
```

Comment:
wasn't this to be removed ?

Reply:
Ah sorry, missed that that one also used it. I removed this commit